klyshko-provisioner/kii-provisioner-run.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/klyshko.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Fail, if any command fails
+set -e
+
+mkdir -p ~/.cs
+cat <<EOF > cs-config
+{
+  "prime" : 0,
+  "r" : 0,
```

Comment:
Interesting, I would have thought that castor already checks for such impossible configurations 🤔

Reply:
Castor is anyway not using these parameters currently. Will not be relevant here in the future as there is a plan to use a Go castor client as soon as it is available (see #2).